src/app/components/ManualEntry.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+  /**
+   * Handles the click when the delte button is pressed
+   * TODO
+   */
+  const handleDelete = (): void => {
+    console.log("Delete button clicked");
+  };
+</script>
+
+<style>
+  .language__sources-manual-entry {
+    border-radius: 20px;
+    padding: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .language__sources-manual-entry-tablename input {
+    height: 30px;
+    width: 400px;
+    border-radius: 2px;
+    border: 1px solid var(--lite-white);
+    padding: 5px;
+    text-align: center;
+    font: 24px var(--main-font);
```

Comment:
Just a FYI. A global font size variables have been introduced in this pr [#98 ](https://github.com/eddieantonio/predictive-text-studio/pull/131/files/9759ce99024d8cfc87b32a60e1bcf353dfe3a781#diff-7e6d987ee4579aedab388447b754b3ba599ac9051c5c9c16a82cc59367ed64de).